front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/SurveyResultsWidget/SurveyReportFilter.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+// components
+import { Box, Select, Text } from '@citizenlab/cl2-component-library';
+import styled from 'styled-components';
+import ProjectFilter from 'containers/Admin/dashboard/components/filters/ProjectFilter';
+import T from 'components/T';
+
+// hooks
+import usePhases from 'hooks/usePhases';
+import useLocalize from 'hooks/useLocalize';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+
+// typings
+import { IOption } from 'typings';
+
+const StyledSelect = styled(Select)<{ padding?: string }>`
+  ${({ padding }) =>
+    padding
+      ? `
+    select {
+      padding: ${padding};
```

Comment:
Perhaps this shouldn't be necessary... is this for padding inside of the select? The design should use our standard padding I think